### ISSUE TITLE
2020-12-11

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#390]
3215.simple model with proper hyperparameters can outperform; (2020-12-3)
3226.there may be a huge space for improvement in hyper-parameter tuning; (2020-12-3)
323II.The Model
3241.distributional models for KBC have recently been extensively studied; (2020-12-3)
3252.achieved better empirical results with the same model by changing hyper-parameters of the training procedure; (2020-12-3)
3263.DistMult score; (2020-12-3)
3274.normalizes the scores by a softmax function; (2020-12-3)
328III.Experiments
3291.Datasets; WN18 and FB15k; (2020-12-3)
